src/main/scala/cromwell/engine/RuntimeAttribute.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.engine
+
+object RuntimeAttribute extends Enumeration {
+  type RuntimeAttribute = Value
+  val failOnStderr = Value("failOnStderr")
+  val docker = Value("docker")
+  val serialize = Value("serialize")
+  val memory = Value("memory")
+}
```

Comment:
There's also the issue that some of these are codified and some are arbitrary for backend.

@cjllanwarne had a great suggestion that `Backend` has a `supportedRuntimeAttributes` or something like that. However I don't think the right way to handle that is with one monolithic enum, e.g. if there are 97 different backends and only one uses a particular runtime attribute it shouldn't be in that enum, but I'd still want it to be in that supported attributes construct.